pkg/build/generator/generator.go

```diff
@@ -261,6 +261,16 @@ func (g *BuildGenerator) Instantiate(ctx kapi.Context, request *buildapi.BuildRe
 	}
 	glog.V(4).Infof("Build %s/%s has been generated from %s/%s BuildConfig", newBuild.Namespace, newBuild.ObjectMeta.Name, bc.Namespace, bc.ObjectMeta.Name)
 
+	// Add the BuildConfig to the Build's OwnerReference, so that it will be
+	// deleted when the BuildConfig is deleted
+	// TODO: is there a utility function for generating an owner reference from an object?
+	newBuild.OwnerReferences = append(newBuild.OwnerReferences, kapi.OwnerReference{
+		APIVersion: "v1",          // BuildConfig.APIVersion is not populated
```

Comment:
not exactly, but we did learn that "v1" is not the right value :)

i think the net was there isn't a good answer today because the value is supposed to be the name of the api group for builds, but builds aren't in an api group today.  I think @deads2k implied that emptystring, or "core" or some other value could be used for things that aren't in an apigroup, but i'm not sure if that's true today, or something that he's hoping to make happen in the GC implementation.